cognee/modules/pipelines/operations/run_tasks.py

```diff
@@ -159,15 +161,18 @@ async def run_tasks_base(tasks: list[Task], data = None, user: User = None):
             })
             raise error
 
-async def run_tasks_with_telemetry(tasks: list[Task], data, pipeline_name: str):
+async def run_tasks_with_telemetry(tasks: list[Task], data, pipeline_name: str, config: dict):
+    logger.debug("\nRunning pipeline with configuration:\n%s\n", json.dumps(config, indent = 1))
+    
     user = await get_default_user()
 
     try:
         logger.info("Pipeline run started: `%s`", pipeline_name)
         send_telemetry("Pipeline Run Started", user.id, {
             "pipeline_name": pipeline_name,
-        })
-
+        } | config
+        )
```

Comment:
This formatting looks like there is some issue with the code. :) Maybe move the closing bracket `)` up with `config`.

Reply:
done